src/main/java/bio/terra/service/filedata/flight/ingest/ValidateIngestFileLoadModelStep.java

```diff
@@ -0,0 +1,31 @@
+package bio.terra.service.filedata.flight.ingest;
+
+import bio.terra.model.FileLoadModel;
+import bio.terra.service.job.JobMapKeys;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import bio.terra.stairway.exception.RetryException;
+
+public class ValidateIngestFileLoadModelStep implements Step {
+  @Override
+  public StepResult doStep(FlightContext context) throws InterruptedException, RetryException {
+    FlightMap inputParameters = context.getInputParameters();
+    FileLoadModel loadModel =
+        inputParameters.get(JobMapKeys.REQUEST.getKeyName(), FileLoadModel.class);
+
+    if (!loadModel.getTargetPath().startsWith("/")) {
```

Comment:
Can this check be done at initial request time, in `IngestRequestValidator`? It seems like it would be better to error earlier / quicker if possible.

Reply:
It turns out we can't do this because the `FileLoadModel` may be generated inside the flight itself, vs being specified in the original web API request.